socorro/cron/jobs/matviews.py

```diff
@@ -68,16 +68,6 @@ class SignaturesCronApp(_MatViewBackfillBase):
     depends_on = ('reports-clean',)
 
 
-class TCBSCronApp(_MatViewBackfillBase):
-    proc_name = 'update_tcbs'
```

Comment:
If we're not removing them in this PR, can we write up a follow-up bug to do it?